scripts/test/e2e/run

```diff
@@ -15,7 +15,11 @@ function fetch_images {
 
 function setup {
     local project=$1
-    COMPOSE_PROJECT_NAME=$1 COMPOSE_FILE=$2 docker-compose up --build -d >&2
+    local file=./e2e/compose-env.yaml
+
+    DOCKERD_EXPERIMENTAL="${DOCKERD_EXPERIMENTAL:-}"
+    test -n "${DOCKERD_EXPERIMENTAL}" && file="${file}:./e2e/compose-env.experimental.yaml"
```

Comment:
The calling code uses `DOCKERD_EXPERIMENTAL=0`, which also passes the test above -- not what you wanted I think?

Reply:
:man_facepalming: